### PR TITLE
issue #11

### PR DESCRIPTION
**Pull request by jakon89**
Hi, I implemented idea from issue #11.
All test passed:
https://travis-ci.org/jakon89/FeedBundle
Backwards compatibility is maintained.

I also added PHPUnit4 as composer dependency. It's much convenient to use with PHPstorm.

I look forward to your feedback.

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -40,10 +40,24 @@ public function getConfigTreeBuilder()
                     ->requiresAtLeastOneElement()
                     ->useAttributeAsKey('name')
                     ->prototype('array')
-                        ->children()
+						->children()
                             ->scalarNode('title')->isRequired()->end()
                             ->scalarNode('description')->isRequired()->end()
-                            ->scalarNode('link')->isRequired()->end()
+							->arrayNode('link')
```
Comment (eko) [inline DependencyInjection/Configuration.php]: Can you please update all the tabs with 4 spaces in order to be compliant with the PSR-2 code style please?

```
Code MUST use 4 spaces for indenting, not tabs.
```

See: http://www.php-fig.org/psr/psr-2/

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -40,10 +40,24 @@ public function getConfigTreeBuilder()
...
+								->isRequired()
+								->beforeNormalization()
+									->ifString()
+									->then(function ($v) { return [ 'direct' => $v]; })
```
Comment (eko) [inline DependencyInjection/Configuration.php]: Can you please remove the short array annotation with the following please (in order to be compliant with PHP < 4.0.0):

```php
->then(function ($v) { return array('direct' => $v); })
```

File: DependencyInjection/Configuration.php [hunk 3]
```diff
@@ -40,10 +40,24 @@ public function getConfigTreeBuilder()
...
+								->end()
+								->children()
+									->scalarNode('route_name')->end()
+									->arrayNode('route_params')
+										->useAttributeAsKey('key')
+										->prototype('scalar')->end()
+									->end()
+									->scalarNode('direct')->end()
```
Comment (eko) [inline DependencyInjection/Configuration.php]: I suggest giving the `direct` configuration name a more understanding name such `uri`. I think it should be better, what do you think?

File: Feed/Feed.php [hunk 4]
```diff
@@ -104,9 +104,29 @@ public function set($parameter, $value)
      */
     public function get($parameter, $default = null)
     {
+		if('link' == $parameter){
```
Comment (eko) [inline Feed/Feed.php]: In order to follow the Symfony code conventions, can you please update the line to add spaces after `if` statement and before opening bracket?

```php
if ('link' == $parameter) {
```

**Comment by eko** [comment]
Hi @jakon89,

Thank you for working on this feature.

However, I've added some comments in order to keep a readable code.

Can you please fix them?

Thank you!

**Comment by jakon89** [comment]
"Can you please fix them?"
Yes, no problem.

I will fix this PR in this week. Thanks

**Comment by jakon89** [comment]
Hi,
Should be ok now :)

**Comment by eko** [comment]
One more thing, are you familiar with PHPunit's tests in order to add a unit test for the `$feed->get('link')` different cases?

If not, I will add it by myself later.

Thank you @jakon89

**Comment by jakon89** [comment]
Sure, I can :)
I can clone ```Eko\FeedBundle\Tests\Feed\FeedManagerTest``` with new configuration.
Provide you solution how to write test if you want.
We have to cover two cases ( link as string and as array) so I don't know where to put new test.

**Comment by jakon89** [comment]
I don't know how to refactor test, would be great to see you in action :)

**Comment by eko** [comment]
Thank you @jakon89, I will add a unit test later.

**Pull request merged**